source/documentation/submit-draft-movement.html.md.erb

```diff
@@ -1,10 +1,43 @@
 ---
-title: Submit draft movement
-weight: 20
+title: Submit new draft movement
+weight: 30
 ---
 
-# Submit draft movement
+# Submit a new draft excise movement
 
-Information about the endpoint will appear here, for example:
+Use the POST endpoint [Submit a new draft excise movement]() to create a new excise movement. 
 
-Use the POST endpoint [for messages relating to the ERN]() with the messageType "Draft" and the messageBody "IE815" to submit a draft movement to the customs office. This request is also called a ‘Submitted Draft of E-AD/E-SAD’ (IE815) message. 
+This request is also called a ‘submit draft movement’ (IE815) message. 
+
+## Successful response
+
+If you use this endpoint to send a small message (up to 10MB), a successful response will contain:
+
+- a movement ID
+- a consignor ID
+- a local reference number (LRN)
+- a status
+
+It may also contain an administrative reference code (ARC).
```

Comment:
I thought that the submit draft move is an async process. By that I mean if you their is an error you get an error  response back straight away, but if the trader is successful he/she will get a 202 (acceped) code back and then the trader would need to pull for the ARC.

Reply:
@mauro-fanton could be! The yaml file had a response schema (attached screenshot) that included these items so I included them in the service guide. But I did think it looked like dummy data - would be strange to have a 'Cancelled' response to a draft movement submission, for example.

It depends how accurate we want the information to be for alpha / user testing. I could include a line at the top that makes it clear this text doesn't reflect the finished API and will change as development progresses.

<img width="745" alt="Screenshot 2023-08-18 at 13 49 57" src="https://github.com/hmrc/emcs-api-service-guide/assets/93519425/7ae54d7b-9958-4f53-b8b7-630105769c03">